deps/wazuh_testing/wazuh_testing/data/qactl_conf_validator_schema.json

```diff
@@ -0,0 +1,367 @@
+                          "type": "string"
+                        },
+                        "wazuh_install_path": {
+                          "type": "string",
+                          "default": "/var/ossec"
+                        }
+                      },
+                      "allOf": [
+                        {
+                          "if": {
+                            "properties": {"type": {"const": "sources"}}
+                          },
+                          "then": {"required": ["wazuh_branch"]},
+
+                          "else": {"required": ["local_package_path"]}
+                        },
+                        {
+                          "if": {
+                            "properties": {"target": {"const": "agent"}}
+                          },
+                          "then": {"required": ["manager_ip"]}       
+                        }        
+                      ]
+                    }
+                  }
+                },
+                "qa_framework": {
+                  "type": "object",
+                  "required": [
+                    "wazuh_qa_branch",
+                    "qa_workdir"
+                  ],
+                  "properties": {
+                    "wazuh_qa_branch": {
+                      "type": "string"
+                    },
+                    "qa_workdir": {
+                      "type": "string"
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "tests": {
+      "type": "object",
+      "patternProperties": {
+        "^host[0-9]*$": {
+          "type": "object",
+          "required": ["host_info", "test"],
+          "properties": {
+            "host_info": {
+              "type": "object",
+              "required": [
+                "connection_method",
+                "host",
+                "user",
+                "connection_port",
+                "ansible_python_interpreter"
+              ],
+              "properties": {
+                "connection_method": {
+                  "type" : "string"
+                },
+                "host": {
+                  "type" : "string"
+                },
+                "user": {
+                  "type": "string"
+                },
+                "password": {
+                  "type": "string",
+                  "default": "empty"
+                },
+                "ssh_private_key_file_path":{
+                  "type": "string"
+                },
+                "connection_port": {
+                  "type": "integer"
+                },
+                "ansible_python_interpreter": {
+                  "type": "string"
+                }
+              },
+              "if": {
+                "properties": {"password": {"const": "empty"}}
+              },
+              "then": { "required": ["ssh_private_key_file_path"]}
+            },
+            "test": {
```

Comment:
```suggestion
            "^test[0-9]*": {
```

Reply:
After talking about it, this suggested change will be omitted and postponed for future modifications.